index.js

```diff
@@ -169,3 +169,5 @@ function isPossibleNode(node) {
     return false;
   }
 }
+
+module.exports = Plugin;
```

Comment:
In general, I think we'd prefer to put this above at the class definition...

(made an inline suggestion)

Reply:
Sounds good. I originally kept the __broccoliFreatures__ on the prototype which required me to have the export like this but since I moved it to the constructor this makes more sense.